src/sections/tickets.tsx

```diff
@@ -0,0 +1,149 @@
+import Section from '../future-hopr-lib-components/Section';
+import { useAppDispatch, useAppSelector } from '../store';
+import { useEffect, useRef, useState } from 'react';
+import { actionsAsync } from '../store/slices/node/actionsAsync';
+import { utils } from '@hoprnet/hopr-sdk';
+const { APIError } = utils;
+
+function TicketsPage() {
+  const dispatch = useAppDispatch();
+  const tickets = useAppSelector((selector) => selector.sdk.tickets);
+  const statistics = useAppSelector((selector) => selector.sdk.statistics);
+  const loginData = useAppSelector((selector) => selector.auth.loginData);
+  const [redeemSuccess, set_redeemSuccess] = useState(false);
+  const [redeemErrors, set_redeemErrors] = useState<
+    { status: string | undefined; error: string | undefined }[]
+  >([]);
+  const [showStatistics, set_showStatistics] = useState(false);
+  const [showTickets, set_showTickets] = useState(false);
+
+  useEffect(() => {
+    if (loginData.apiEndpoint && loginData.apiToken) {
+      dispatch(
+        actionsAsync.getStatisticsThunk({
+          apiEndpoint: loginData.apiEndpoint,
+          apiToken: loginData.apiToken,
+        })
+      );
+      dispatch(
+        actionsAsync.getTicketsThunk({
+          apiEndpoint: loginData.apiEndpoint,
+          apiToken: loginData.apiToken,
+        })
+      );
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I would add the needed parameters here, as they seem to be static and will not trigger useEffect